### PR TITLE
Break action in channels API

### PR DESCRIPTION
**Pull request by tesarm**
I've added break action into channels api which allows anyone to go to the next step in callflows.
Usage is same as hangup action

### CODE REVIEW

File: core/whistle_apps/src/whapps_call_command.erl [hunk 1]
```diff
@@ -838,6 +839,20 @@ b_echo(Call) ->
 %%--------------------------------------------------------------------
 %% @public
 %% @doc
+%% Produces the low level wh_api request to break the channel.
+%% This request will execute immediately
+%% @end
+%%--------------------------------------------------------------------
+-spec break(whapps_call:call()) -> 'ok'.
+break(Call) ->
+  Command = [{<<"Application-Name">>, <<"hangup">>}
```
Comment (lazedo) [inline core/whistle_apps/src/whapps_call_command.erl]: `<<"hangup">>` ?

**Comment by lazedo** [comment]
@tesarm @revgen-michal looks good except the formatting that you are trying to merge. please follow the existing indentation, don't change code that you're not touching.

**Comment by fenollp** [comment]
Also squashing this history afterwards would be nice

**Comment by tesarm** [comment]
few fixes made. It should be all right now. 

BTW what IDE do you use for erlang? I use intellij idea but I have completely different code formatting.

**Comment by lazedo** [comment]
@tesarm @revgen-michal much better, thank you. please rebase from latest master and squash your commits

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,6 @@ core/whistle/mime.types
 
 applications/*/priv/archive_data/*
 
+*.iml
+.idea/
```
Comment (fenollp) [inline .gitignore]: This should be part of your gitignore on your $HOME

**Comment by tesarm** [comment]
I somehow fucked up my git repo, so I've forked it again and pushed all changes in one commit.
- new pull request submitted though : https://github.com/2600hz/kazoo/pull/1847

**Pull request closed without merging**